sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicDataOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Dynamic
+{
+    /// <summary>
+    /// </summary>
+    public class DynamicDataOptions
+    {
+        internal static readonly DynamicDataOptions Default = new()
```

Comment:
"Ted nit": putting my @tg-msft hat on, he tends to prefer and has pushed for more read-only property accessors:
```suggestion
        internal static DynamicDataOptions Default { get; } = new()
```

Reply:
Oh cool, happy to change that.  Do you know why?

Reply:
Not really, no. I brought up once during a review that the IL for loading a field vs. calling a property is less/smaller, but Ted pointed out the optimized code path is the same. I guess maybe it looks better?

Reply:
Isn't there a difference between how the compiler treats the property? I believe a property with only a getter is effectively equivalent to:

```c#
internal static DynamicDataOptions Default { get; private set;} = new()
```

But `readonly` adds enforcement that it must be assigned during construction and prevents even private setting of the value.